.chloggen/fix_red-metrics.yaml

```diff
@@ -0,0 +1,41 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use new default metrics namespace/prefix for span RED metrics in Jaeger query.
+
+
+# One or more tracking issues related to the change
+issues: [1072]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  Use the new RED metrics default namespace `traces.span.metrics` for retrieval from Prometheus.
+  Since OpenTelemetry Collector version `0.109.0` the default namespace is active.
+  The namespace taken into account by jaeger-query can be configured via a TempoStack CR entry.
+  To achieve this the Operator will set the jaeger-query `--prometheus.query.namespace=` flag.
+  Since Jaeger version 1.62, jaeger-query uses `traces.span.metrics` as default too.
+
+  Example how to overwrite the default namespace by configuring it in the CR:
```

Comment:
Can we make this example for collector before 0.109.0 ? I think the majority of users will want to know this.

@frzifus did you test that `redMetricsNamespace` can be set to an empty string?

Reply:
I can confirm that 
```
apiVersion: tempo.grafana.com/v1alpha1
kind:  TempoStack
metadata:
  name: redmetrics
spec:
  storage:
    secret:
      name: minio-test
      type: s3
  storageSize: 1Gi
  template:
    gateway:
      enabled: false
    queryFrontend:
      jaegerQuery:
        enabled: true
        monitorTab:
          enabled: true
          prometheusEndpoint: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
          redMetricsNamespace: ""
        ingress:
          type: route
```

Leads to an empty flag value:
```
      - args:
        - --query.base-path=/
        - --span-storage.type=grpc
        - --grpc-storage.server=localhost:7777
        - --query.bearer-token-propagation=true
        - --prometheus.query.namespace=
        - --prometheus.tls.enabled=true
        - --prometheus.token-file=/var/run/secrets/kubernetes.io/serviceaccount/token
        - --prometheus.token-override-from-context=false
        - --prometheus.tls.ca=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
        env:
        - name: METRICS_STORAGE_TYPE
          value: prometheus
        - name: PROMETHEUS_SERVER_URL
          value: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091
        image: docker.io/jaegertracing/jaeger-query:1.62.0
        imagePullPolicy: IfNotPresent
        name: jaeger-query
```